src/engine/controls/loopingcontrol.cpp

```diff
@@ -1431,6 +1443,18 @@ void LoopingControl::slotBeatJump(double beats) {
     }
 }
 
+void LoopingControl::slotBeatJumpSizeHalve(double pressed) {
+    if (pressed != 0) {
```

Comment:
```suggestion
    if (pressed > 0) {
```
This is also an issue in the functions you took as template. 
We need to consider to fix it all or leave this untouched.

Reply:
I changed it for the four affected controls